axelrod/player.py

```diff
@@ -122,6 +186,17 @@ def set_match_attributes(self, length=-1, game=None, noise=0):
         self.match_attributes = {"length": length, "game": game, "noise": noise}
         self.receive_match_attributes()
 
+    def set_seed(self, seed):
+        """Set a random seed for the player's random number generator."""
+        if seed is None:
+            warnings.warn(
+                "Initializing player with seed from Axelrod module random number generator."
+                " Results may not be seed reproducible.")
```

Comment:
I think we need a space here because the strings get concatenated, so the sentences will run together otherwise. I'll move it to the end of the first string so it doesn't look weird.